web/src/components/Dashboard.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+
+class Dashboard extends Component {
```

Comment:
Should we follow Class-based components or pure function-based w/ react hooks?

Reply:
We should follow function based components. Code refactored